sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/Options.cs

```diff
@@ -20,9 +14,6 @@ public class Options
         [Option('i', "clientId", Required = true, HelpText = "Client Id of the application Id to login, or the application Id used to log the user in.")]
         public string ClientId { get; set; }
 
-        [Option('m', "loginMethod", Required = false, Default = "AppId", HelpText = "Choose between: AppId, User.")]
```

Comment:
Should we include a way to influence the bool to the DefaultAzureCredential ctor?

Reply:
We should if it works. Will be easy to add in if it works for you. I've not been successful yet.